gallery/src/styles/NxCard/NxCardVerticalLayoutExample.tsx

```diff
@@ -0,0 +1,266 @@
+              XXX%
+            </div>
+            <div className="nx-card__text">
+              City advert motion apophenia film skyscraper sentient beef noodles industrial grade motion tanto BASE jump
+              girl realism
+            </div>
+          </div>
+          <footer className="nx-card__footer">
+            <a href="#" className="nx-text-link">Film sprawl wristwatch math-engine</a>
+          </footer>
+        </section>
+      </div>
+
+      <div className="nx-card-container nx-card-container--row">
+        <section className="nx-card" aria-label="card 7 in row">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Header</h3>
+          </header>
+          <div className="nx-card__call-out">
+            42
+          </div>
+        </section>
+      </div>
+      <div className="nx-card-container nx-card-container--row">
+        <section className="nx-card" aria-label="card 8 in row">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Card header</h3>
+          </header>
+          <div className="nx-card__call-out">
+            <NxFontAwesomeIcon icon={faShapes} className="nx-icon--card-call-out" />
+          </div>
+          <footer className="nx-card__footer">
+            <a href="#" className="nx-text-link">Link</a>
+          </footer>
+        </section>
+      </div>
+      <div className="nx-card-container nx-card-container--row">
+        <section className="nx-card nx-card--align-left nx-card--75wide" aria-label="card 9 in row">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">A wide card with left aligned content</h3>
+          </header>
+          <div className="nx-card__content nx-card__content--2-column-example">
+            <div className="nx-card__call-out">
+              <NxFontAwesomeIcon icon={faChartLine} className="nx-icon--card-call-out" />
+            </div>
+            <div className="nx-card__text">
+              Lots of content to force the card to a wider size sentient noodles sprawl tower corrupted cartel
+              carbon decay sentient post- singularity
+            </div>
+          </div>
+        </section>
+        <section className="nx-card" aria-label="card 1 in row">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Card header</h3>
+          </header>
+          <div className="nx-card__content nx-card__content--2-column-example">
+            <div className="nx-card__call-out">
+              <NxFontAwesomeIcon icon={faShapes} className="nx-icon--card-call-out" />
+            </div>
+            <div className="nx-card__text">
+              Sentient post-singularity
+            </div>
+          </div>
+        </section>
+      </div>
+      <div className="nx-card-container nx-card-container--row">
+        <section className="nx-card nx-card--50wide" aria-label="card 10 in row">
```

Comment:
Is there an intent that these two would fit side-by-side? They don't

Reply:
> Is there an intent that these two would fit side-by-side? They don't

Fixed here though it feels clunky: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/8f5da678aea8c9dbaeb0dae27bd601b2b3379311

Reply:
> Fixed here though it feels clunky: 8f5da67

That's now how I'd do it, I'd have a class for equal-width cards (probably on the container, not the cards themselves), which would set `flex: 1; max-width: auto` on the cards.

Reply:
> > Fixed here though it feels clunky: [8f5da67](https://github.com/sonatype/sonatype-react-shared-components/commit/8f5da678aea8c9dbaeb0dae27bd601b2b3379311)
> 
> That's now how I'd do it, I'd have a class for equal-width cards (probably on the container, not the cards themselves), which would set `flex: 1; max-width: auto` on the cards.

I have tweaked the examples here: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/2bfda09a9ab7682780dc05476eaea76a98e7581b

`max-width: auto` is not valid. I've used `unset` instead. The issue with having the cards appear side-by-side was `flex-wrap: wrap` which always favoured wrapping. I created a custom class to handle this and tweaked the examples further here: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/367617f0ee1bbccfefecb3a81c23913471eb2aeb and https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/26d7fd068a9f7201927fd0283b3911d5ab00b543

Reply:
> The issue with having the cards appear side-by-side was flex-wrap: wrap which always favoured wrapping

So that's only an issue because you specified `width: 50%`.  What I suggested above was to not do that, and to instead just give them the same flex factor.

Reply:
> > The issue with having the cards appear side-by-side was flex-wrap: wrap which always favoured wrapping
> 
> So that's only an issue because you specified `width: 50%`. What I suggested above was to not do that, and to instead just give them the same flex factor.

Ah, my mistake lay in not applying both `shrink` and `grow`. I liked this method so much I created a modifier class for it and some further examples: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/f7f3dee65e5f1b5d4edb6166456e6886e6fd0e41